sandstone/lib/tests/python/test_filewatcher.py

```diff
@@ -0,0 +1,78 @@
+import unittest
+import mock
+from sandstone.lib.filewatcher import Filewatcher
+import tempfile
+import os
+from sandstone.apps.filebrowser.posixfs import PosixFS
+import time
+import copy
+
+
+class FilewatcherTestCase(unittest.TestCase):
+    def setUp(self):
+        self.test_dir = tempfile.mkdtemp()
+
+    def tearDown(self):
+        # remove all watches
+        dirs = copy.deepcopy(Filewatcher._watches)
+        for key, value in dirs.items():
+            Filewatcher.remove_directory_to_watch(key)
+        # remove temp directory
+        PosixFS.delete_file(self.test_dir)
```

Comment:
Use `shutil` or something similar to clean up after the test methods, instead of a component of the software under test.